### PR TITLE
Open admin on strapi start

### PR DESCRIPTION
**Pull request by Aurelsicoko**
My PR is a: 💅 Enhancement

Main update on the: Admin

When the server starts, it opens automatically the administration panel and reuses the same tab instead of opening a new one. 

This behaviour can be disabled with `server.json` configuration file.

### CODE REVIEW

**Comment by lauriejim** [comment]
I tried in `dev` mode. I clone, setup and create new app.
When I start my project, it open the admin on port `4000` but I haven't start the admin so I have an error `This site can’t be reached`.

Can we detect if the admin is running on port `4000`. And depend on, open the admin on `1337` or `4000`

**Comment by soupette** [comment]
@Aurelsicoko @lauriejim we can base the window opening on checking if wether or not we generated the app with the `--dev` flag?

**Comment by Aurelsicoko** [comment]
Done guys, it should be okay 👍

**Comment by derrickmehaffy** [comment]
@Aurelsicoko mayyyyy want to check that strapi isn't running on a headless system or at least add some kind of config option to turn this off/on (I recommend off by default)

**Comment by derrickmehaffy** [comment]
@Aurelsicoko I mean you have strapi running on a server that doesn't have a GUI and you want to access the adminUI remotely during development (think virtual machines). It shouldn't attempt to open a browser window and should be accessible from outside localhost. (allowing for easier development)

**Comment by Aurelsicoko** [comment]
I think we can already merge that work because it works most of the time. Also, there is an option to disable the `autoOpen` (see https://github.com/strapi/strapi/pull/1727/files#diff-6f6d5033152dc11eb49ce482f93ff8a8R139)

File: packages/strapi/lib/core/configurations.js [hunk 1]
```diff
@@ -321,9 +322,13 @@ module.exports.app = async function() {
   // default settings
   this.config.port = get(this.config.currentEnvironment, 'server.port') || this.config.port;
   this.config.host = get(this.config.currentEnvironment, 'server.host') || this.config.host;
+  this.config.url = `http://${this.config.host}:${this.config.port}`;
```
Comment (derrickmehaffy) [inline packages/strapi/lib/core/configurations.js]: Baddddd, really shouldn't be hard coding this even in development :stuck_out_tongue:

Comment (lauriejim) [reply packages/strapi/lib/core/configurations.js]: Check two lines before :) based on env

**Pull request merged**